projects/map-ol/src/lib/map-ol.service.spec.ts

```diff
@@ -352,6 +355,13 @@ const beforeEachFn = () => {
     name: 'ukis group',
     layers: [ukisVectorLayerJson, ukisWmtsLayer, ukisWmsLayer, ukisRasterLayer, ukisCustomLayerVector]
   });
+
+  ukisMergeLayer = new Layer({
+    id: 'ID-ukis-merge-layer',
+    name: 'ukis merge layer',
+    type: 'custom',
```

Comment:
It could also be discussed it type custom is best for merged layers or we should create the in a other function with another type?

Reply:
as we discussed on the phone, maybe type: 'merged' would be cleaner. Alternative: new constructor for this kind of layers e.g. new MergedLayer()... because here we have additional [optional] parameter "mergedLayers" which is mandatory for this new 'merged' layer... and in custom layer there is another one - 'custom_layer'... using generic constructor (new Layer()) for both of them is a bit confusing

Reply:
one more idea, maybe to call it LayerStack instead of merged? (again, not absolutely necessary.. just an idea)

Reply:
Then I would propose this

```
export class StackedLayer extends Layer implements IStackedLayerOptions {
  type = 'stacked';
  layers: Layer[] = [];
  constructor(options: IStackedLayerOptions) {
    super(options as ILayerOptions);
    Object.assign(this, options);
  }
}
```

We already have `Layer`, `RasterLayer`, `VectorLayer` , `CustomLayer`, `WmtsLayer`, `WmsLayer` and `LayerGroup`